rotkehlchen/chain/ethereum/airdrops.py

```diff
@@ -418,33 +192,33 @@ def check_airdrops(
             }:
                 amount = token_normalized_value_decimals(int(amount), 18)  # type: ignore
             if addr in addresses:
-                asset = airdrop_data[1]
+                asset_identifier = airdrop_data.asset_identifier
```

Comment:
Check the amount deserialization here. I believe we need a note in the data repo to use always the already decimal precision form of the asset. Take notes of that. Maybe would be a good moment to remove that if case but in another PR